thisdot/lets-chat-with
# [Chore]: Add Contributors List to the README

# Overview

README changes that include a contributors list.

# Screenshots

## Before

N/A since it did not exist 😛

## After

<img width="710" alt="Screen Shot 2023-02-15 at 10 22 35 AM" src="https://user-images.githubusercontent.com/4316134/219119114-4ba5e463-48c3-4fdf-b8f3-d61f68b66f05.png">

# Details

## General

- Added a contributor list to the README.

## Automated Testing

N/A

### Manual Testing

1. Scroll down to the bottom of the root README and ensure the contributors table is formatted properly.
